### PR TITLE
design home page

### PR DESCRIPTION
**Pull request by wowKimsohyun17**
## Related Tickets:
- [#Ticket ID](https://edu-redmine.sun-asterisk.vn/issues/30259)

## WHAT this PR does?
home page

## Notes Impacted Areas:
- N/A.

## Attachments:
2s

### CODE REVIEW

**Comment by wowKimsohyun17** [comment]
ready

File: resources/lang/vi/label.php [hunk 1]
```diff
@@ -18,4 +18,9 @@
     'contact_mail' => 'Nhập email để đăng ký nhận những thông tin hữu ích về học tập từ F8',
     'enter_email' => 'Email của bạn',
     'register' => 'Đăng ký',
+    'description_banner' => 'Học lập trình với các khóa học chất lượng để đi làm!',
+    'title_banner' => 'CỘNG ĐỒNG HỌC LẬP TRÌNH MIỄN PHÍ',
+    'change_theme' => 'Giao diện sáng tối',
+    'settings' => 'Cài đặt',
+    'log_out' => 'Đăng xuất',
```
Comment (huyenttm-0901) [inline resources/lang/vi/label.php:25]: viết file en đi em

**Review by huyenttm-0901 — COMMENTED** [review]
(commented)

**Comment by wowKimsohyun17** [comment]
ready

**Pull request merged**